app/scripts/modules/google/src/loadBalancer/configure/ssl/listener.html

```diff
@@ -23,9 +23,11 @@
               </ui-select>
             </td>
             <td>
-              <input class="form-control input-sm"
-                     readonly
-                     required ng-model="ctrl.loadBalancer.portRange"/>
+              <select class="form-control input-sm"
```

Comment:
This doesn't seem to work properly - if regardless of the value I select here, the resulting frontend's port is 443. This is not a UI bug (I verified that the payload leaving Deck is sending the right port), but this should be resolved before merging this PR.